spec/truefi2/BorrowingMutex.spec

```diff
@@ -23,13 +31,53 @@ rule functionDoesNotBanUnlockedBorrower() {
     require isReasonableEnv(e);
     calldataarg args;
     if (f.selector == lock(address, address).selector) {
-        address locker;
+        address locker = anyReasonableAddress();
         address lockee;
-        require isReasonableAddress(locker);
         lock(e, lockee, locker);
     } else {
-        sinvoke f(e, args);
+        f(e, args);
```

Comment:
+1

For context: Shelly told us that `sinvoke` is default behavior and can be omitted unless we're doing fancy tricks.